src/utils.jl

```diff
@@ -498,6 +498,14 @@ function make_fillrange_from_ribbon(kw::KW)
     kw[:fillrange] = make_fillrange_side(y, rib1), make_fillrange_side(y, rib2)
 end
 
+#turn tuple of fillranges to one path
```

Comment:
You're right, I thought maybe it would be useful to also implement ribbon for other backends, but I guess it can stay in the plotly file as long as it's only used by plotly.